apps/interpreter/src/parse-only.spec.ts

```diff
@@ -38,13 +79,25 @@ describe('Parse Only', () => {
   });
 
   it('should exit with 1 on error', async () => {
+    const validModel = (await fs.readFile(pathToValidModel)).toString();
+
+    tempFile = path.resolve(
+      os.tmpdir(),
+      // E.g. "0.gn6v6ra9575" -> "gn6v6ra9575.jv"
+      Math.random().toString(36).substring(2) + '.jv',
+    );
+
+    // Write a partial valid model in that file
```

Comment:
It took me a few minutes to understand what this test is actually doing which I don't think is a good sign. Instead of 'partial valid model' I would make this explicit: 'Create an invalid model by truncating a previously valid model'. 

I think overall this is way overcomplicated though, why do you not just create an actually invalid model file and load it?